libc/socket/unix-socket.c

```diff
@@ -405,9 +405,15 @@ void unix_msg_data_only(int type)
 	close(fd[1]);
 }
 
-TEST(test_unix_socket, unix_msg_data_only)
+
+TEST(test_unix_socket, unix_stream_sock_send_data)
```

Comment:
Not that test case? From what I see, these should be changed:  `unix_msg_data_and_fd` & `unix_msg_fork` :

Zynq-qemu
![image](https://github.com/phoenix-rtos/phoenix-rtos-tests/assets/116297248/a3864b1c-1e8c-4feb-9bfc-21be4e58b2cd)

stm32:
![image](https://github.com/phoenix-rtos/phoenix-rtos-tests/assets/116297248/e6baa681-078e-4fdb-89a2-95697a77de66)